apollo-server/v3/integration-testing/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "types": ["jest", "node"],
```

Comment:
I think this is going to lead to jest types being valid in non-test code too. I think you want to have a second tsconfig.test.json which pulls in the jest types, and use the tsConfig option to ts-jest in jest.config.js to point at it? (Or you can avoid a second file and put the compilerOptions directly in the ts-jest section of jest.config.js apparently?)

Reply:
Went with your suggestion on slack and created a separate `tsconfig.json` just for testing which included the "jest" type.